src/openqaoa-core/qaoa_components/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
```

Comment:
I would call this folder just 'components' instead of 'qaoa_components', if at some point there is another vqa, a lot of files inside this folder will be used

Reply:
I believe `qaoa_components` is a better name since the parameterizations make more sense in the context of QAOA than VQAs.

I think keep the module name as it is for now is fine.

@Q-lds @vishal-ph Quick comments here would help. Thanks.

Reply:
+1 shahidee